src/http/get-media-000image/index.js

```diff
@@ -26,9 +26,25 @@ exports.handler = response(async req => {
   if (!originalImage) {
     return reply.text('Not Found', { statusCode: 404 });
   }
+  // set up optimization params
+  // must be at least min image size:
+  const width = req.queryStringParameters.w > config.minimumImageSize.width ?
+    req.queryStringParameters.w : config.minimumImageSize.width;
+  const height = req.queryStringParameters.h > config.minimumImageSize.height ?
+    req.queryStringParameters.h : config.minimumImageSize.height;
+  // default is 'resize'
+  const type = req.queryStringParameters.resize || false;
+  // set params based on whetehr resizing, covering or containing:
+  const params = { width, height, type };
+  // determine new file name:
+  if (type === 'resize') {
+    const imageBaseTokens = imageName.split('.');
+    const newBaseName = imageBaseTokens.slice(0, imageBaseTokens.length - 1).join('.');
+    imageName = `${newBaseName}_${type}_${width}_${height}.${imageBaseTokens[imageBaseTokens.length - 1]}`;
+  }
   // optimize it:
   try {
-    const imageBuffer = await optimize({ quality: [config.quality, config.quality] }, originalImage.Body);
+    const imageBuffer = await optimize.resize(params, originalImage.Body);
```

Comment:
should we be resizing if we don't pass in w/h?  Looks like we lost optimize

Reply:
If we don't pass in a w+h, do we want to just store the unmodified image in 'optimized' folder?

Reply:
we should optimize the image no matter what